README.md

```diff
@@ -65,7 +65,7 @@ To include a screenshot, please generate the output using the [screenshotTable.s
 
 **For screenshot consistency, please have your font set to 13pt Monaco and no transparency on the window**
 
-It would also be very helpful if you `cd tools/` and run `python3 update_all.py` to generate all formats of your scheme
+It would also be very helpful if you `cd tools/` and run `python3 gen.py` to generate all formats of your scheme
```

Comment:
Please also show how to correctly install the dependencies needed for this to run; I can see that you've included the `requirements.txt`, but speaking as someone that has come to mistrust python packaging in general, I think it would be great if you could show eg: the correct virtualenv invocation needed to run this without requiring installing any system packages.

Reply:
I was planning to add a section on creating new templates after merge (in case something changes during the review). Alongside I will add information about the setting of the environment.